git-merge-pr

```diff
@@ -100,9 +100,10 @@ if [ $CURRENT_BRANCH = $BRANCH_TO_MERGE ] &&  [ $PRUNE_FLAG = off ]; then
 fi
 
 echo "Stashing any local changes and checking out remote branch 'origin/$DESTINATION_BRANCH'"
-STASH_OUTPUT="$(git stash save -a)" || exit 1;
+STASH_OUTPUT="$(git -c commit.gpgSign=false stash save -a)" || exit 1;
 
 echo "Securely fetching remote branches and revisions"
+echo "GPG signature required to commit RSL fetch entry"
```

Comment:
Do we want to preface this somehow so it's offset from otherwise normal progress?
`<--- GPG signature required...`
`\o/ GPG signature required...`
`<-----> GPG signature required...`

Reply:
Sounds like we do